crates/hc_sandbox/tests/cli.rs

```diff
@@ -227,3 +222,15 @@ fn get_sandbox_command() -> Command {
         Err(_) => Command::from(std::process::Command::cargo_bin("hc-sandbox").unwrap()),
     }
 }
+
+async fn get_launch_info(stdout: ChildStdout) -> LaunchInfo {
+    let mut lines = BufReader::new(stdout).lines();
+    while let Ok(Some(line)) = lines.next_line().await {
+        if let Some(index) = line.find("#!0") {
```

Comment:
is it always correct that we want the info from the first conductor?

Reply:
For tests launching a single conductor yes. If somebody adds a test to launch multiple conductors then they'd need to add some logic here to wait for a given number of launch infos to appear